packages/js/product-editor/src/hooks/use-number-input-props.ts

```diff
@@ -34,6 +37,11 @@ export const useNumberInputProps = ( {
 				onFocus( event );
 			}
 		},
+		onBlur( event ) {
+			if ( onBlur ) {
+				onBlur( event );
+			}
+		},
 		onKeyUp( event: React.KeyboardEvent< HTMLInputElement > ) {
```

Comment:
I think we also want to do the following...

```
onKeyDown( event: React.KeyboardEvent< HTMLInputElement > ) {
			if ( event.code === 'ArrowUp' || event.code === 'ArrowDown' ) {
				event.preventDefault();
			}
		},
```

Otherwise, when the keydown for `ArrowUp` occurs, the browser moves the cursor to the beginning of the input field before we then increment the number and move the cursor back to the end of the field. It looks odd, and I am concerned that this back-and-forth for the cursor could be an accessibility issue.